pkg/engine/datasource/rest_datasource/rest_datasource_test.go

```diff
@@ -259,11 +259,11 @@ func TestFastHttpJsonDataSourcePlanning(t *testing.T) {
 						Variables: resolve.NewVariables(
 							&resolve.ContextVariable{
 								Path:     []string{"idVariable"},
-								Renderer: resolve.NewPlainVariableRendererWithValidation(`{"type":"string"}`),
+								Renderer: resolve.NewPlainVariableRendererWithValidation(`{"type":["string"]}`),
 							},
 							&resolve.ContextVariable{
 								Path:     []string{"a"},
-								Renderer: resolve.NewPlainVariableRendererWithValidation(`{"type":"string"}`),
+								Renderer: resolve.NewPlainVariableRendererWithValidation(`{"type":["string","null"]}`),
```

Comment:
@jensneuse I couldn't figure out how nullability is determined for rest parameters.  It seems like the last param is nullable and the other ones aren't?  I modified all the tests with that rule in mind, in any case.